### ISSUE TITLE
Error loading torchvision.dll

### ISSUE THREAD

**Issue by matobler** [TabakM/CameraTrapDetectoR#7]
I am getting an error that the torchvision.dll can not be loaded (the file downloaded and is present in the Cache folder). Same error on two different machines. Visual C++ was installed. R version is 4.1.3, 64bit. Any suggestions?

`Error in inDL(x, as.logical(local), as.logical(now), ...) : 
  unable to load shared object 'C:/Users/Mathias/AppData/Local/CameraTrapDetector/CameraTrapDetector/Cache/torchvision.dll':
  LoadLibrary failure:  The specified procedure could not be found.`

**Comment by mikeyEcology** [comment]
Yes. There was an update to the `torch` package that requires me to update this package. I'm working on it but it will take a few days. In the meantime, you can use this 
```
devtools::install_version("torch", version = "0.6.0", repos = "http://cran.us.r-project.org")
```
to install the older version of torch, which should work.

**Comment by matobler** [comment]
That worked, thanks a lot for the quick response! Is GPU supported with this version or will it only run on the CPU (a bit slow at the moment for large datasets). I have the GPU version of Torch installed and CUDA is working but your model is not using the GPU. I have no experience with Torch (use TF for most of my work), so I might be missing something.
